src/uuid/builder/uuidbuilder.rs

```diff
@@ -16,7 +16,7 @@ use std::convert::Into;
 use std::sync::atomic::{AtomicU64, Ordering};
 use std::time::{SystemTime, UNIX_EPOCH};
 
-use crate::uprotocol::Uuid as uproto_Uuid;
+use crate::uprotocol::uuid::UUID as uproto_Uuid;
```

Comment:
One thing I thought of here... looking to hear your thoughts. I don't know what the convention is, but if I'm looking for impls of UUid or its usage within this crate, I think it makes it a little harder to find due to this `use foo as bar`.

WDYT? Other than that, I'm good with the changes.

Reply:
I have removed the alias in the latest commit I have pushed ...